examples/complete/main.tf

```diff
@@ -0,0 +1,50 @@
+module "atlas_basic" {
+  source = "../../"
+  org_id = var.org_id
+
+  project_name = "complete-example-project"
+  ip_addresses = ["1.2.3.4", "6.7.8.9"]
```

Comment:
should we also use vars here?

Reply:
I don't think so. Other module examples use values directly. ([aws-vpc](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/examples/simple/main.tf), [security group](https://github.com/terraform-aws-modules/terraform-aws-security-group/blob/master/examples/complete/main.tf))
I think this is meant to be copy-pasted into the users .tf files.